.github/workflows/unittests.yml

```diff
@@ -77,34 +27,21 @@ jobs:
         run: larq_compute_engine/tflite/build_make/build_lce.sh --native --pip
       - name: "Install TF Lite Python Runtime Package"
         run: pip3 install /tmp/tflite_pip/python3/dist/*.whl
-      - name: "Python Test Dependency: Build Bazel Package Creator Binary"
-        run: ./bazelisk build :build_pip_pkg
-      - name: "Python Test Dependency: Build LCE PIP Package"
-        run: bazel-bin/build_pip_pkg artifacts
-      - name: "Python Test Dependency: Install LCE PIP Package"
-        run: pip3 install larq pytest pytest-xdist artifacts/*.whl
-      - name: "Run Python Tests"
-        run: |
-          # Go to a different directory to force it to use the installed package
-          # and not use the local larq_compute_engine directory
-          cd larq_compute_engine/tflite
-          python3 -m pytest . -n auto
```

Comment:
We should refactor those tests to use the new converter at some point.

Reply:
:+1:  And probably just use the pre-built pip package instead of recompiling the entire converter.